src/test/resources/scenario43/output.json

```diff
@@ -0,0 +1,156 @@
+{
+    "timestamp": "2015-05-01T00:00:00.000+02",
+    "matches": [
+      {
+        "system_id": 100,
+        "subscription_id": 2,
+        "product_id": 1299,
+        "cents": 50,
+        "confirmed": true
+      },
+      {
+        "system_id": 100,
+        "subscription_id": 2,
+        "product_id": 1300,
+        "cents": 50,
+        "confirmed": true
+      },
+      {
+        "system_id": 100,
+        "subscription_id": 1,
+        "product_id": 1625,
+        "cents": 50,
+        "confirmed": true
+      },
+      {
+        "system_id": 100,
+        "subscription_id": 1,
+        "product_id": 1625,
+        "cents": 100,
+        "confirmed": false
+      },
```

Comment:
2 times the same tripple of system_id, subscription_id and product_id but with different cents. Why?
I think 1 item with 100 cent would be correct here.

Reply:
Similar as in the case above. These matches are part of different matching scenarios:
- the match with `100` cents would be used if **only** the product `1625` was assigned to the host
- the pair of matches with `50` cents is used if both `1625` and `1878` are assigned to the host.

Reply:
Here the situation is complicated with the inherited virtualization subscription, but the result is correct. There are 2 situations, which both produce their own sets of matches in the output and these are same on the first glance.

The confirmed match comes from the situation, where the subscription `1` (unlimited virt.) covers both `1625` and `1878` on the guests and together with this, the (inherited virt.) subscription `2` covers the guests as well.

The unconfirmed match comes from the situation, where the subscription `1`  covers both `1625` and `1878` on the guests, BUT the subscription `2`is not confirmed. Btw, the number of the `cents` is irrelevant for unconfirmed matches, maybe we can talk about it.